mautic/mautic
# M3: Refactor  lookup_id field

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR contains code that enchances functionality of the `lookup_id` field.
It's now possible to use this field not only for owner_id data, but actually for any type of data.


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
This is not a bug so there is nothing to reproduce.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a segment, add a `owner` filter to that segment. Make sure you are able to select owners from the dropdown.
